### ISSUE TITLE
Add Grafana

### ISSUE THREAD

**Issue by Ornias1993** [jailmanager/jailman#9]
After having added InfluxDB it would be totally awesome to add Grafana too..
Considering some users have trouble setting up Grafana on FreeNAS to begin with...

**Comment by Ornias1993** [comment]
PR's to include this can now be PR'ed agaist the jails/grafana branch.

**Comment by Ornias1993** [comment]
While Adding grafana isn't that much of a problem.
It would be prefered if the influxdb jail would be available first, to have a supporting backend in.

**Comment by Ornias1993** [comment]
Unblocked, due to inclusion of influxDB on the dev branch

**Comment by Ornias1993** [comment]
@colemamd I see you are already working on it, if you still plan to submit it (please do), mind if I set this on "in progress" and assign you to it?

**Comment by colemamd** [comment]
Go ahead. I was just waiting until I can get it to auto-connect to a Data Source. But i can go ahead and submit it then pr it later to refine.

**Comment by Ornias1993** [comment]
You can Send in a Draft-PR anyway ;)

**Comment by Ornias1993** [comment]
Btw you should've the rights now to assign yourself too :)

**Comment by Ornias1993** [comment]
Closed due to being merged into dev
